apps/nextjs/src/app/api/cron-jobs/expired-exports/route.ts

```diff
@@ -54,14 +54,13 @@ async function updateExpiredAtAndDelete(fileIds: string[]) {
       log.error(`Error processing file with gdriveFileId: ${id}`, error);
       failedIds.push(id);
     }
-
-    if (failedIds.length > 0) {
-      const errorMessage = `Failed to process the following file IDs: ${failedIds.join(
-        ", ",
-      )}`;
-      log.error(errorMessage);
-      throw new Error(errorMessage);
-    }
+  }
+  if (failedIds.length > 0) {
+    const errorMessage = `Failed to process the following file IDs: ${failedIds.join(
+      ", ",
+    )}`;
+    log.error(errorMessage);
+    throw new Error(errorMessage);
```

Comment:
Could this potentially be 1000 IDs? I don't know if there's a limit on an error message length, but I'm happy to go with it and we'll see if it needs another tweak or not